src/iris/ui/__init__.py

```diff
@@ -399,6 +421,9 @@ def init(config, app):
     auth_module = config.get('auth', {'module': 'iris.ui.auth.noauth'})['module']
     auth = importlib.import_module(auth_module)
     auth_manager = getattr(auth, 'Authenticator')(config)
+    qr_base_url = config['qr_base_url']
```

Comment:
We should change this (and the qr endpoint) to be optional, since it has no meaning if mobile isn't configured.

Reply:
Implemented

Reply:
Makes sense, implemented.